tests/conftest.py

```
@@ -117,6 +117,59 @@ class MockRequestsExceptions:
+        self._current_nodeid: str = ""
+        self._real_connect = sqlite3.connect
+        self._patched: bool = False
+
+    def install(self) -> None:
+        if self._patched:
+            return
+        real_connect = self._real_connect
+        registry = self._registry
+        tracker = self
+
+        def _tracked_connect(*args: Any, **kwargs: Any) -> sqlite3.Connection:
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)